gammapy/image/utils.py

```diff
@@ -260,11 +282,13 @@ def exclusion_distance(exclusion):
 
     Parameters
     ----------
-    TODO
+    exclusion : array_like
```

Comment:
At the moment some image utils functions specify the input as `array_like`, some as `~numpy.ndarray`.

The main difference is that functions that say they accept `array_like` should also accept Python lists like `[[1,2], [3,4]]` as input and then you usually have to call
```
exclusion = np.asanyarray(exclusion)
```
at the top.

I don't think this is useful ... usually users have data as ndarrays and if they don't they can call np.asanyarray themselves. I've written up this coding style guideline for Gammapy here:
https://gammapy.readthedocs.org/en/latest/development/index.html#what-checks-and-conversions-should-i-do-for-inputs

So please change to inputs
```
~numpy.ndarray
```
everywhere in this file and if there are calls to `np.asanyarray` at the start of the functions remove them.

This is tedious maintenance / polishing work, I know. But that's just the reality ... Gammapy already contains quite a bit of mediocre code with no-good docs and tests, and one of your jobs for GSoC would be to improve this for the parts of the code you work on. But there's also the aspect that you'll implement new features ... that's the fun part.